docs/operate/validators/run-validators.mdx

```diff
@@ -61,14 +108,16 @@ Like the local setup, there are a few base arguments you should provide when con
 | `--originChainName`                             | Name of the chain being validated. For example: `ethereum`.                                             |
 | `--chains.[originChainName].customRpcUrls`      | Override the default RPC URLs used by the Validator for your origin chain.                              |
 | `--chains.[originChainName].blocks.reorgPeriod` | Number of block confirmations the Validator needs to wait for before signing the `Mailbox` merkle root. |
+| `--chains.[originChainName].customGrpcUrls`     | Configure multiple gRPC endpoints for Cosmos chains, similar to `customRpcUrls`. |
```

Comment:
similarly might be nice to link to the full reference on these and to put these right under `customRpcUrls`

Reply:
got it - moved these under the rpc config